### PR TITLE
#27500 Prepare Vault module Tests for PHPUnit 8

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Preparing PHPUnit Tests to run with PHPUnit 8.
PRs for each module is separated to make tests easier to merge.

### Fixed Issues (if relevant)
1. magento/magento2#27500

### Manual testing scenarios (*)
N/A

### Questions or comments
PR #27701 needs to be merged first (otherwise PHPUnit tests fail)

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Vault/Test/Unit/Model/PaymentTokenRepositoryTest.php [hunk 1]
```diff
@@ -48,49 +48,49 @@ class PaymentTokenRepositoryTest extends \PHPUnit\Framework\TestCase
...
 
     /**
-     * @var \Magento\Vault\Model\ResourceModel\PaymentToken\Collection|MockObject collection
+     * @var Collection|MockObject collection
      */
     protected $collectionMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Vault/Test/Unit/Model/PaymentTokenRepositoryTest.php]: Please import this class

File: app/code/Magento/Vault/Test/Unit/Model/Ui/Adminhtml/TokensConfigProviderTest.php [hunk 2]
```diff
@@ -526,7 +525,7 @@ private function getTokenUiComponentProvider($token)
      * @param mixed $value
      * @param int $atIndex
      *
-     * @return \PHPUnit_Framework_MockObject_MockObject
+     * @return \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Vault/Test/Unit/Model/Ui/Adminhtml/TokensConfigProviderTest.php]: Please import this class

File: app/code/Magento/Vault/Test/Unit/Model/Ui/Adminhtml/TokensConfigProviderTest.php [hunk 3]
```diff
@@ -553,7 +552,7 @@ private function createExpectedFilter($field, $value, $atIndex)
      * @param int $customerId
      * @param int $entityId
      * @param string $vaultProviderCode
-     * @return \PHPUnit_Framework_MockObject_MockObject
+     * @return \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Vault/Test/Unit/Model/Ui/Adminhtml/TokensConfigProviderTest.php]: Please import this class

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7419](https://jira.corp.magento.com/browse/ENGCOM-7419) has been created to process this Pull Request

**Pull request merged**